run_pplm.py

```diff
@@ -375,7 +402,6 @@ def get_bag_of_words_indices(bag_of_words_ids_or_paths: List[str], tokenizer) ->
             words = f.read().strip().split("\n")
         bow_indices.append(
             [tokenizer.encode(word.strip(),
-                              add_prefix_space=True,
```

Comment:
This line is required. Each line in BOW needs a prefix, without which, `bow_indices` is correct. This drastically changes the output of BOW model output due to incorrect gradient computation.

Reply:
Additionally, new L241 needs
```
output_hidden_states=True,
return_dict=False
```